### DISCUSSION TITLE
"<bevy game> has control of your pointer.  Press Esc to take back control." - how to disable?

### DISCUSSION THREAD

**Discussion by stlhoosier97** [aevyrie/bevy_mod_picking#238] [Q&A]
Hello!  I am using bevy_mod_picking and of course it works great in the executable.  When I build a .wasm and run it on Firefox, I get this pop-up every time I select something and it causes some amount of erroneous inputs if I don't hit Esc each time.

You can see it in action here:
https://stlhoosier97.github.io/index.html

(Hit 'press to start' and then choose a white piece with your mouse).

I have looked everywhere but cannot see how to disable this.  Can anyone help?

**Comment by aevyrie** [comment]
This plugin does not capture the cursor. I believe this is a setting in a bevy window resource.
